### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by SiddharthV1**
This PR is to make the Layout responsive

### CODE REVIEW

File: V2/tezex/src/theme.tsx [hunk 1]
```diff
@@ -45,11 +45,11 @@ const theme = createTheme({
...
+  palette: {
+    grey: {
       main: purple[500],
-		}
-	},
+    }
+  },
   typography: {
     fontFamily: "Inter",
    },
```
Comment (vishakh) [inline V2/tezex/src/theme.tsx:55]: Best to delete old code.

**Review by vishakh — COMMENTED** [review]
Left a minor comment. Could you please also include a screencap or screenshot in the PR description showing the changes?

File: V2/tezex/src/theme.tsx (hunk 1 above)
Comment (SiddharthV1) [reply V2/tezex/src/theme.tsx:55]: Will do , its still a draft started work on it today .

Comment (vishakh) [reply V2/tezex/src/theme.tsx:55]: Yes, realized it's in draft only after submitting the "review". :clown_face:

**Pull request merged**